tests/requests/test_base_http_lilya.py

```diff
@@ -248,22 +247,6 @@ async def app(scope: Any, receive: "Receive", send: "Send") -> None:
     assert response.json() == {"json": "Receive channel not available"}
 
 
-async def test_request_disconnect() -> None:  # pragma: no cover
```

Comment:
Why removing this test?

Reply:
because the test is not executed and the below it tests the same but with a different exception.

It maybe would be helpful to enable the warnings

Reply:
I disabled not to overwhelm the log. Let it be for now. Got it.